### ISSUE TITLE
Azure Account Extension can't sign in to sovereign clouds

### ISSUE THREAD

**Issue by cxznmhdcxz** [microsoft/vscode-azure-account#221]
OS: Windows
VS Code version: 1.48.2(user setup)

I found Azure Account Extension can't sign in to sovereign clouds.
And even causes command palette to crash.

Repro steps:
1. Install extensions: Azure Account, and Azure Storage (for convenience to check whether signed in)
1. Open command palette, enter command "azure: sign in to azure cloud"
![image](https://user-images.githubusercontent.com/47708215/92426759-50818580-f1bd-11ea-8f24-2a0188f9b249.png)
1.  There SHOULD be a list of azure clouds to choose, including Azure, Azure China, Azure Gov, and Azure Germany.
But NOW I get nothing. There is no list.
![image](https://user-images.githubusercontent.com/47708215/92427223-8a06c080-f1be-11ea-837b-b30505d46e05.png)
1. At this point, press "Esc" after step3 to close command palette, and then re-open command palette.
Now whatever command you type in, press "Enter" or click on the command, it won't be executed, nothing will happen.

**Comment by RMacfarlane** [comment]
Duplicate of https://github.com/microsoft/vscode-pull-request-github/issues/1936

**Comment by RMacfarlane** [comment]
Ah, sorry, obviously wrong issue. https://github.com/microsoft/vscode-azure-account/issues/216
